src/m_menu.c

```diff
@@ -3308,6 +3309,8 @@ static void M_UpdateCrosshairItems (void)
 {
     DISABLE_ITEM(!hud_crosshair, stat_settings2[stat2_xhairhealth]);
     DISABLE_ITEM(!STRICTMODE(hud_crosshair), stat_settings2[stat2_xhairtarget]);
+    DISABLE_ITEM(!STRICTMODE(hud_crosshair && hud_crosshair_target),
```

Comment:
Why does "lock on target" depend on "highlight on taget"?

Reply:
Technically speaking, I'm pretty sure the features can work independently of each other. However, with the current code, the `linetarget` search code block will run only if `hud_crosshair_target == true`. I could add `|| hud_crosshair_lockon` to effectively make the feature independent, just let me know.

Reply:
Yes, please. I was a bit puzzled why I couldn't have one without the other.

Reply:
Heh, it appears that one of the conditions for the `HU_UpdateCrosshairLock()` call in `R_ProjectSprite()` was `hud_crosshair_target` instead of `hud_crosshair_lockon`, so technically speaking the target lock-on _was_ dependent of the target highlight up until now. Oh well, that seems to be fixed now.